gradle.properties

```diff
@@ -1 +0,0 @@
-org.gradle.jvmargs=--illegal-access=permit
```

Comment:
I think thymeleaf upgrade means that using groovy 3 instead of groovy 2 which removes the reflective access warning and thus not needed